bin/propolis-server/src/lib/vm/state_driver.rs

```diff
@@ -211,6 +217,8 @@ impl InputQueue {
                 let mut guard = self.inner.lock().unwrap();
                 if let Some(guest_event) = guard.guest_events.pop_front() {
                     return InputQueueEvent::GuestEvent(guest_event);
+                } else if let Some(req) = guard.buffered_external.pop_front() {
```

Comment:
Nit: the docstring should be updated to also include the `buffered_external` queue.

Reply:
Fixed in bfb8b07.